### ISSUE TITLE
Support for filament PHP

### ISSUE THREAD

**Issue by sacalito** [absszero/vscode-laravel-goto#21]
for some reason in filament does not work 
```

<?php

namespace App\Filament\Pages\Admin\Deploy;

use Filament\Pages\Page;
use App\Models\Users\User;

class Deploy extends Page {
    protected static ?string $navigationIcon = 'heroicon-o-code-bracket';
    protected static string $view = 'filament.pages.admin.deploy.deploy';

    private $user;

    public function mount(): void{
        $this->user = User::find(auth()->user()->id);
        abort_unless($this->user->hasAnyRole(['admin']), 403);
    }
    
    public static function shouldRegisterNavigation(): bool {
        /** @var \App\Models\Users\User */
        $user = auth()->user();
        return $user->hasAnyRole(['admin']);
    }
}
```

**Comment by sacalito** [comment]
hi, 
im really glad you answered!

your plugin makes my work 1000% faster

ill give all the info you need!

for now i installed pilamentphp v3 and as of now there are two diferent times the extension does not recognize

in any view, for example 

```
<x-layouts.public.layout :title="'Cobrar'">
	<livewire:public.discount>
</x-layouts.public.layout>
```

![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/712c64dd-8378-4d3b-8103-a8cba5f304d6)
 none of the livewire components are recognized
 
 
in the controller the code that works is

`view('pages.public.charge');`

but with fillament the views are like so 

`protected static string $view = 'filament.pages.admin.email.email';`

![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/8fa9765b-a9d2-42f5-a8f5-578387aead7b)

i hope this info help, 

just to thankyou again, your plugin makes working with laravel so much much faster

one more note 
 in the view itseld where i embed the livewire component my work arround was to put comented line with the path 
 
 
![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/766206e4-542d-4c50-9343-8d847c730f9a)

for some reason the controller path works awsomly but the resources path does not work at all just, when right clicking on the comented line and clicking laravel go to ir just open the pallete with norhitng writen on it

![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/2e62dd0a-e7f6-45d0-a6a0-1d362ca602cd)

if you need any other info please let me know

**Comment by absszero** [comment]
Hi, I published a new version 4.11.0 on [Visual Studio marketplace](https://marketplace.visualstudio.com/items?itemName=absszero.vscode-laravel-goto)

those paths should work now

```blade
<x-layouts.public.layout :title="'Cobrar'">
    <livewire:public.discount>
</x-layouts.public.layout>

{{-- "resources/views/pages/public/charge" --}}
```

```php
protected static string $view = 'filament.pages.admin.email.email';
```

**Comment by sacalito** [comment]
WOW thank you very very much!

i cant tell you how much this helps in laravel development!

**Comment by sacalito** [comment]
im really gratefull for all the changes you did.

i just found other two places where the view is not recognized

in a Mailable class

```
class TestMail extends Mailable
{
    use Queueable, SerializesModels;

    /**
     * Create a new message instance.
     */
    public function __construct(public User $user)
    {
    }

    /**
     * Get the message envelope.
     */
    public function envelope(): Envelope
    {
        return new Envelope(
            subject: 'Another test email',
        );
    }

    /**
     * Get the message content definition.
     */
    public function content(): Content
    {
        return new Content(
            view: 'emails.test',
            //html: 'emails.test',
            //text: 'emails.test',
        );
    }

    /**
     * Get the attachments for the message.
     *
     * @return array<int, \Illuminate\Mail\Mailables\Attachment>
     */
    public function attachments(): array
    {
        return [];
    }
}
```

![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/ccb9ac21-18b5-4050-a3c0-78715dc2f045)

and also in the web.php file for routes /routes/web.php
```

/******* Public ********/
        Route::view('/', 'pages.public.index');
        Route::view('/rifas', 'pages.public.rifas');
        Route::view('/descuentos', 'pages.public.descuentos');
```

![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/3192c049-66c0-4268-b0ec-21455393b356)

in both cases when i do `Alt + ;`    i get the pallete empty

![image](https://github.com/absszero/vscode-laravel-goto/assets/62297560/23946609-80d8-4e1c-8133-0a7f86cfc414)


i can even make a test repository for you to make it easier and not copy and paste the codes

for the Mailable class you can just run 

`php artisan make:mail OrderShipped`

this will create a file on `/app/Mail/OrderShipped` with the default mail class 

```
<?php

namespace App\Mail;

use Illuminate\Bus\Queueable;
use Illuminate\Contracts\Queue\ShouldQueue;
use Illuminate\Mail\Mailable;
use Illuminate\Mail\Mailables\Content;
use Illuminate\Mail\Mailables\Envelope;
use Illuminate\Queue\SerializesModels;

class OrderShipped extends Mailable
{
    use Queueable, SerializesModels;

    /**
     * Create a new message instance.
     */
    public function __construct()
    {
        //
    }

    /**
     * Get the message envelope.
     */
    public function envelope(): Envelope
    {
        return new Envelope(
            subject: 'OrderShipped',
        );
    }

    /**
     * Get the message content definition.
     */
    public function content(): Content
    {
        return new Content(
            view: 'view.name',
        );
    }

    /**
     * Get the attachments for the message.
     *
     * @return array<int, \Illuminate\Mail\Mailables\Attachment>
     */
    public function attachments(): array
    {
        return [];
    }
}
```

again you'r plugin is super awsome thanks

**Comment by absszero** [comment]
thank you for your help, I published a new version [4.12.0 ](https://marketplace.visualstudio.com/items?itemName=absszero.vscode-laravel-goto)
